web/beacon-app/src/components/common/QuickView/QuickView.tsx

```diff
@@ -54,10 +55,10 @@ const QuickView: React.FC<any> = ({ data }) => {
   }, [data]);
 
   return (
-    <div className="grid grid-cols-2 gap-y-10 gap-x-5 lg:grid-cols-4">
+    <div className="grid grid-cols-2 gap-10 lg:grid-cols-4">
       {getValidData().map((item, index) => (
-        <QuickViewCard key={item.name} title={capitalize(item.name)} color={BRAND_COLORS[index]}>
-          {item.value}
+        <QuickViewCard key={item.name} title={STAT_NAME[index]} color={BRAND_COLORS[index]}>
```

Comment:
Instead of changing the backend, I thought that using a map here for the stat's name would make it easier in case it's ever changed in the future. I noticed a TODO about the `getValidData` function so thought this would be the better solution for now.

Reply:
While it's an excellent suggestion, it's important to consider that the names received from the backend should serve as our primary reference. This approach helps us prevent inconsistencies in naming and ensures that everything remains up to date, minimizing the risk of oversight.